tests/artifacts/assets/results.csv

```diff
@@ -0,0 +1 @@
+1,2,3,4
```

Comment:
Did you check in this file just so the folder will exist? I don't really see it being used anyway.

Reply:
this is being used in the tests take a look at `test_log_artifact`